<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_org">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gray">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>92</width>
      <height>112</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="captureBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="trainBtn">
    <property name="geometry">
     <rect>
      <x>678</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recognitionBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>recognition</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>eigenFace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fisherFace</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LBPHFace</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="trainSVMBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>470</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>train SVM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
